packages/astro/src/vite-plugin-astro-server/error.ts

```diff
@@ -24,8 +23,6 @@ export function recordServerError(
 	// Our error should already be complete, but let's try to add a bit more through some guesswork
 	const errorWithMetadata = collectErrorMetadata(err, config.root);
 
-	telemetry.record(eventError({ cmd: 'dev', err: errorWithMetadata, isFatal: false }));
```

Comment:
this is going to remove telemetry for all errors that happen during a request. Do we just want to remove the unhandled rejections? if so we can check that `_err?.name === 'UnhandledRejection'`

Reply:
I didn't know we assigned a specific name to it. Yeah I think that works too 👍